qiskit/transpiler/__init__.py

```diff
@@ -365,9 +444,11 @@
    a scheduling phase can be applied to optionally account for all the idle time in the circuit.
    At a high level the scheduling can be thought of as inserting delays into the circuit to account
    for idle time on the qubits between the execution of instructions. For example, if we start with a
-   circuit such as:
+   circuit such as a ghz circuit and then call :func:`~.transpile` on it with ``scheduling_method``
```

Comment:
Strictly in text, GHZ is an acronym and ought to be capitalised.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9346/commits/2ace0a4550792f326f9151230b6e4df9fb1caa15